content/03-reference/01-tools-and-interfaces/01-prisma-schema/03-generators.mdx

```diff
@@ -4,158 +4,50 @@ metaTitle: 'Generators (Reference)'
 metaDescription: 'Generators in your Prisma schema specify what assets are generated when the `prisma generate` command is invoked. This page explains how to configure generators.'
 ---
 
-## Overview
+<TopBlock>
 
-A generator can be specified via the `generator` block in the Prisma schema. Generators in your Prisma schema specify what assets are generated when the `prisma generate` command is invoked.
+A generator determines which assets are created when you run the `prisma generate` command, including:
 
-## Fields
+* The **client language** (currently only `prisma-client-js` is available)
+* Target **engine binaries** (for example, `debian-openssl-1.0.x` if you are deploying to Ubuntu 14.04, or `native` if you are working locally)
```

Comment:
This is technically only a configuration of the `prisma-client-js` and does not apply to other generators.

Reply:
Updated to say specifically `prisma-client-js`